packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -631,6 +637,26 @@ export default class NewVm extends BaseComponent {
         )
     }
   )
+
+  _getAutomaticNetworks = () => {
+    const poolNetworks = this._getPoolNetworks()
+
+    return poolNetworks.filter(
+      network => network.other_config && network.other_config.automatic
+    )
+  }
+
+  _findNetwork = () => {
+    const automaticNetworks = this._getAutomaticNetworks()
+
+    return automaticNetworks.length !== 0
+      ? automaticNetworks[0]
```

Comment:
We want them all, not only the first one.

Reply:
When adding a template, there I add all the automatic networks. https://github.com/vatesfr/xen-orchestra/blob/e2393202c091a5b11714f768d1f59810d94e6d0b/packages/xo-web/src/xo-app/new-vm/index.js#L504

This method is called only when an interface is added. So we only need a network. But I'm going to rework it.